prow/jobs_config/knative/serving.yaml

```diff
@@ -183,13 +183,13 @@ jobs:
     requirements: [perf]
     command: [runner.sh, ./test/presubmit-tests.sh, --run-test, ./test/performance/performance-tests.sh]
 
-  # - name: certmanager-integration-tests
-  #   types: [presubmit]
-  #   command: [runner.sh, ./test/e2e/certmanager/presubmit-tests.sh, --integration-tests]
+  - name: certmanager-integration-tests
+    types: [presubmit]
+    command: [runner.sh, ./test/e2e/certmanager/presubmit-tests.sh, --run-test, ./test/e2e/certmanager/e2e-tests.sh]
```

Comment:
Let's see how that goes. Gradually we should move the script to the top dir and remove any extra sciprts.

Reply:
Can you make an issue for that?